website/docs/fundament/logo.mdx

```diff
@@ -1,41 +0,0 @@
----
-sidebar_position: 1
----
-
-# Logo
-
-Het logo van het Omgevingsloket wordt gebruikt in digitale diensten om de site te identificeren als een officiële dienst van het Omgevingsloket.
-
-Het logo bestaat uit een beeldmerk én een woordmerk. Het beeldmerk is het zogenoemde "target". Het woordmerk, de naam van de afzender, staat er rechts van. Het target bestaat uit 3 lagen, in 3 kleuren. Het woordmerk bestaat uit 2 kleuren. Optioneel staat er een extra tekstregel onder het woordmerk met de naam van de specifieke dienst, bijvoorbeeld "beheerportaal".
-
-![Details logo](/docs/images/logo-afmetingen.png)
-
-## Hoe te gebruiken
-
-- Plaats het logo altijd linksboven op de website.
-- De hoogte van het logo valt binnen het verticale 8px grid, zie voorbeelden hieronder.
-- Extra tekstregel (optioneel)
-- Extra tekstregel font Asap Regular size 20, line height 24 in grijs (`#666666`);
-  - Max. 20 tot 25 karakters in extra tekstregel;
-  - De tekstregel lijnt links aan de "O" van "Omgevingsloket".
-- Het webfont Asap is beschikbaar onder Open Font License via [Google Fonts](https://fonts.google.com/specimen/Asap).
-- Het logo verwijst altijd naar de homepage van de dienst zelf. AIs er geen homepage beschikbaar is, zoals bij een enkel formulier, dan verwijst het logo naar omgevingsloket.nl
-
-## Beschikbare varianten
-
-|                      | Logo                                                                                                                                                                             | Logo met Beta label                                                                                                                                                                                  |
-| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **Omgevingsloket**   | [![Logo Omgevingsloket](/docs/logos/omgevingsloket.svg "Logo Omgevingsloket")](/docs/logos/omgevingsloket.svg)                                                                   | [![Logo Omgevingsloket Beta](/docs/logos/omgevingsloket-beta.svg "Logo Omgevingsloket Beta")](/docs/logos/omgevingsloket-beta.svg)                                                                   |
```

Comment:
Deze assets staan nog in de repository, graag ook verwijderen.

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/7a96d54e-f22a-4c10-931c-18a2ac4af390)